setup.py

```diff
@@ -493,6 +494,9 @@ def pxd(name):
                     'pxdfiles': ['_libs/src/util', '_libs/lib'],
                     'depends': tseries_depends,
                     'sources': npdt_srces},
+    '_libs.tslibs.np_datetime': {'pyxfile': '_libs/tslibs/np_datetime',
+                               'depends': tseries_depends[:2],
```

Comment:
I would rather be explict here rather than depend on the ordering (you can also make a ``ctseries_depends`` or whatever and use that too).